Classes/Utility/SegmentControl+Border.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  SegmentControl+Border.swift
+//  Freetime
+//
+//  Created by Ehud Adler on 3/12/19.
+//  Copyright © 2019 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+extension UISegmentedControl {
+    func removeBorders() {
+        setBackgroundImage(imageWithColor(color: backgroundColor ?? .white), for: .normal, barMetrics: .default)
+        setBackgroundImage(imageWithColor(color: tintColor ?? .white), for: .selected, barMetrics: .default)
+        setDividerImage(imageWithColor(color: UIColor.clear), forLeftSegmentState: .normal, rightSegmentState: .normal, barMetrics: .default)
+    }
+
+    // create a 1x1 image with this color
+    private func imageWithColor(color: UIColor) -> UIImage {
```

Comment:
Do we have this somewhere else already by chance?

Reply:
Ill search